packages/engine-test-utils/src/__tests__/engine-server/markdown/dendronPub.spec.ts

```diff
@@ -202,20 +210,93 @@ describe("GIVEN dendronPub", () => {
   });
 
   describe("WHEN publish and private hierarchies", () => {
-    const fname = fnameBeta;
-    const config = genPublishConfigWithPublicPrivateHierarchies();
+    let config: StrictConfigV5;
+
+    beforeEach(() => {
+      config = genPublishConfigWithPublicPrivateHierarchies();
+    });
+
+    describe("AND WHEN noteref of published note", () => {
+      let resp: VFile;
 
-    describe("AND WHEN noteRef", () => {
-      describe("AND WHEN noteref of published note", () => {
-        let resp: VFile;
+      beforeAll(async () => {
```

Comment:
the reason you are seeing test issues is because `beforeAll` runs before `beforeEach`. you'll want to move the config initialization into the beforeAll block